locations/linked_data_parser.py

```diff
@@ -161,11 +161,10 @@ def parse(response, wanted_type, parse_json5=False) -> GeojsonPointItem:
         if ld_item:
             item = LinkedDataParser.parse_ld(ld_item)
 
-            if item["website"] is None:
-                item["website"] = response.url
-            elif item["website"] == "":
-                item["website"] = response.url
-            elif item["website"][0] == "/":
+            if isinstance(item["website"], list):
```

Comment:
Check that there's something in the list before indexing into it:

```suggestion
            if isinstance(item["website"], list) and item["website"]:
```

Reply:
`parse_ld` always populates `website`, even if it's None. `isinstance(None, list)` works as expected. Can update if you want though.

Reply:
No prob.